src/main/java/org/jabref/gui/preferences/ShowPreferencesAction.java

```diff
@@ -16,6 +17,12 @@ public ShowPreferencesAction(JabRefFrame jabRefFrame, TaskExecutor taskExecutor)
 
     @Override
     public void execute() {
-        new PreferencesDialogView(jabRefFrame).show();
+        PreferencesDialogView preferencesDialogView = new PreferencesDialogView(jabRefFrame);
+        preferencesDialogView.show();
+        for (PreferencesTab tab: preferencesDialogView.getPreferenceTabList().getItems()) {
+            if (tab.getBuilder().getScene() != null) {
+                Globals.getThemeLoader().installCss(tab.getBuilder().getScene(), Globals.prefs);
```

Comment:
We are trying to get rid of Globals, so it would be nice if you could pass the JabRefPreferences object in the constructor.
Pleas also undraft your PR if you think it's ready for review

Reply:
Thank you for your suggestions. I have change the code to implement this and undraft my PR  ^_^